client/app/visualizations/table/Editor/ColumnEditor.jsx

```diff
@@ -0,0 +1,86 @@
+import { map, keys } from 'lodash';
+import React from 'react';
+import PropTypes from 'prop-types';
+import Input from 'antd/lib/input';
+import Radio from 'antd/lib/radio';
+import Checkbox from 'antd/lib/checkbox';
+import Select from 'antd/lib/select';
+import Icon from 'antd/lib/icon';
+
+import { ColumnTypes } from '../utils';
```

Comment:
Btw, how about adding a "Duration" column type @arikfr

Reply:
@ranbena Simething like we have in some tables which shows a time ago?

Reply:
Exactly

Reply:
Great idea, but I think it's better to do a new PR after we merge this one :-)

Reply:
No doubt 👍

Reply:
This can probably be a setting on the Date/Time column type. And yeah: outside the scope.